### DISCUSSION TITLE
Q: Jekyll搭建的，评论区显示不正常，请看链接：https://www.liuweiqing.life/%E6%B5%8B%E8%AF%95/

### DISCUSSION THREAD

**Discussion by 14790897** [walinejs/waline#1940] [Q&A] [answered]
### Describe the bug | 问题描述

显示不正常：https://www.liuweiqing.life/%E6%B5%8B%E8%AF%95/

### Website URL | 问题网站

https://www.liuweiqing.life/%E6%B5%8B%E8%AF%95/

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment, marked as answer]
有个 panel.scss 复写了很多 Waline 的样式导致了异常，样式冲突这种需要用户自行解决，Waline 不负责处理。

**Reply by 14790897** [reply]
大佬，我本地找不到panel.scss，能给个方向吗，然后我可以问gpt4

**Reply by lizheming** [reply]
放到 page__comments 里面去应该就正常了，不会折腾就换个主题
![image](https://user-images.githubusercontent.com/424491/232959440-9cac547f-d975-4b4f-bdf0-c822fb7357a2.png)

**Reply by 14790897** [reply]
大佬，成功了，我之前根据gpt4的提示把代码放在了comments.html最底部，也就是page_comments的外面，原来要放里面啊，感谢
![image](https://user-images.githubusercontent.com/121866954/232960995-84953a8e-5754-49e4-8f11-f3ba409fbd70.png)
